doc/REST-interface.md

```diff
@@ -84,6 +84,13 @@ Returns various state info regarding block chain processing.
 Only supports JSON as output format.
 Refer to the `getblockchaininfo` RPC help for details.
 
+#### Deployment info
+`GET /rest/deploymentinfo/<BLOCKHASH>.json`
```

Comment:
It may be clearer to indicate both endpoints.
```diff
 #### Deployment info
+`GET /rest/deploymentinfo.json`
 `GET /rest/deploymentinfo/<BLOCKHASH>.json`
```